create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE EXTENSION IF NOT EXISTS "pg-crypto";
+
+CREATE TYPE STATUS_COLOR AS ENUM ('GREEN', 'ORANGE', 'RED', 'GREY');
```

Comment:
Lower case strings? That's what the model gives us, and it would avoid a call to `upper`